src/bricks/renderers/document.ts

```
@@ -102,6 +103,7 @@ export class DocumentRenderer extends RendererABC {
       Component: DocumentViewLazy,
       props: {
         body,
+        stylesheets: compact(stylesheets).filter((url) => isValidUrl(url)),
```

I think we want to throw BusinessError on any invalid URLs vs. silently ignoring?